src/renderer/containers/Console/SavedConnectionItem.tsx

```diff
@@ -100,7 +96,7 @@ export const SavedConnectionItem: React.FC<SavedConnectionItemProps> = ({
                 font-size: 14px;
               `}
             >
-              {connection.folderPath}
+              {path.join(connection.folderPath, nickname ?? "")}
```

Comment:
I'm not sure this is a good idea. Wouldn't this mean that you can't set a custom location for the files without it always having the nickname appended? e.g. if i set the `connection.folderPath` to be `/abc/test` it would always be saved to `/abc/test/nickname`? I'm not sure I'm not board with forcing that,

Reply:
I need some better justification about why that is users might not want them sorted by nickname because I can't see any true downsides. If we have a strong justification then my next question is do we want this to be a setting that gets applied to all consoles or individually?

Reply:
I guess for starters, if I choose a folder to be the folder path for the output files, I would expect the files to be output there -- not in a subdirectory. At the moment in this implementation, I don't think that it's clear that it will end up being saved in the subdirectory. You could potentially address this by heavy signposting. But even then, with the current implementation, the file path should be set at a higher level.  i.e. if (as in the example above) you need to do a `path.join()` at the renderer level, something is wrong. The `connection.folderPath` should have already had the nickname appended by the time it reaches that stage.

Personally speaking I would rather give the users options rather than forcing a particular behaviour. Whether that's a global option in settings that says "Save all spectator games into individual folder", or a checkbox when adding a console, or even path templating like the OBS output file path, I really don't mind.